fala/assets-src/scripts/google_tag_manager_data_layer.js

```diff
@@ -0,0 +1,30 @@
+const fala_form = document.querySelector('#fala_questions');
+
+function getCookie(name) {
+  const value = `; ${document.cookie}`;
+  const parts = value.split(`; ${name}=`);
+  if (parts.length === 2) return parts.pop().split(';').shift();
+  return null;
+}
+
+const cookie_value = getCookie('cookiePermission') === 'Allowed';
```

Comment:
Does this course any page flashes of the cookie banner briefly when the DOM is rendering?